src/nasher/utils/nwn.nim

```diff
@@ -171,7 +171,7 @@ proc updateIfo*(dir, bin, args: string, opts: options.Options, target: options.T
       if askIf(fmt"Changing the module's min game version to '{moduleVersion}' could have unintended consequences.  Continue?"):
         ifoJson["Mod_MinGameVer"]["value"] = %moduleVersion
         success("module min game version set to " & moduleVersion)
-  elif moduleVersion.len > 0:
+  else:
```

Comment:
This makes the `else` block run if `moduleVersion` is blank. If you want `isValid` to only run if a version is set, you need to do:
```nim
if moduleVersion.len > 0:
  if moduleVersion.isValid:
    # ...
  else:
    # error
```